tests/submission_test.py

```diff
@@ -33,7 +33,9 @@
 
 from hepdata.ext.opensearch.api import get_records_matching_field
 from hepdata.modules.permissions.models import SubmissionParticipant
-from hepdata.modules.records.api import format_submission, process_saved_file, create_new_version
+from hepdata.modules.records.api import format_submission, process_saved_file, create_new_version, get_record_data_list, \
+    get_related_hepsubmissions, get_related_datasubmissions, get_related_to_this_datasubmissions, get_table_data_list, \
+    get_related_to_this_hepsubmissions
```

Comment:
As per comment on `api.py`, this is a bit of a run-on import statement which may be more readable spread over more lines.